exporter/prometheusexporter/collector.go

```diff
@@ -207,6 +209,40 @@ func (c *collector) convertIntHistogram(metric pdata.Metric) (prometheus.Metric,
 	return m, nil
 }
 
+func (c *collector) convertSummary(metric pdata.Metric) (prometheus.Metric, error) {
+	dataPoints := metric.Summary().DataPoints()
+
+	var cumSum float64
+	var cumCount uint64
+	quantiles := make(map[float64]float64)
+	labelsMap := pdata.NewStringMap()
+	for i := 0; i < dataPoints.Len(); i++ {
+		sdpi := dataPoints.At(i)
+		qv := sdpi.QuantileValues()
+		for j := 0; j < qv.Len(); j++ {
+			qvj := qv.At(j)
+			quantiles[qvj.Quantile()] += qvj.Value()
```

Comment:
Why is this an addition? Quantiles can't be added.

Reply:
Quantiles indeed can’t be added but I wasn’t sure if there
will ALWAYS be exactly one value with the quantile or not — this part wasn’t clear.
Replacement wouldn’t be my immediate fallback
given that then ordering would matter.

Reply:
There should be exactly one per summary metricpoint for a given quantile, otherwise it's invalid exposition.

Reply:
Fixed, thank you!